java/src/jmri/util/TypeConversionUtil.java

```diff
@@ -0,0 +1,321 @@
+package jmri.util;
+
+import java.text.ParseException;
+import java.util.Collection;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import javax.annotation.CheckForNull;
+import jmri.Reportable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Converts between java types, for example String to Double and double to boolean.
+ * 
+ * @author Daniel Bergqvist Copyright 2019
+ */
+public final class TypeConversionUtil {
+    
+    /**
+     * Is this object a Boolean?
+     * @param object the object to check
+     * @return true if the object is a Boolean, false otherwise
+     */
+    public static boolean isBoolean(Object object) {
+        return object instanceof Boolean;
+    }
+    
+    /**
+     * Is this object an integer number?
+     * <P>
+     * The method returns true if the object is any of these classes:
+     * <ul>
+     *   <li>AtomicInteger</li>
+     *   <li>AtomicLong</li>
+     *   <li>BigInteger</li>
+     *   <li>Byte</li>
+     *   <li>Short</li>
+     *   <li>Integer</li>
+     *   <li>Long</li>
+     * </ul>
+     * @param object the object to check
+     * @return true if the object is an object that is an integer, false otherwise
+     */
+    public static boolean isIntegerNumber(Object object) {
+        return (object instanceof java.util.concurrent.atomic.AtomicInteger)
+                || (object instanceof java.util.concurrent.atomic.AtomicLong)
+                || (object instanceof java.math.BigInteger)
+                || (object instanceof Byte)
+                || (object instanceof Short)
+                || (object instanceof Integer)
+                || (object instanceof Long);
+    }
+    
+    /**
+     * Is this object an integer number?
```

Comment:
This comment looks like a copy and paste, as it mentions integers instead of floats

Reply:
Yes, it is. Thanks.